kbswagger.json

```diff
@@ -0,0 +1 @@
"TVD","TWD","TZS","UAH","UGX","USD","UYU","UZS","VEF","VND","VUV","WST","XAF","XCD","XDR","XOF","XPF","YER","ZAR","ZMW","ZWD","BTC"]},"effectiveDate":{"type":"string","format":"date-time"},"processedAmount":{"type":"number"},"processedCurrency":{"type":"string","enum":["AED","AFN","ALL","AMD","ANG","AOA","ARS","AUD","AWG","AZN","BAM","BBD","BDT","BGN","BHD","BIF","BMD","BND","BOB","BRL","BSD","BTN","BWP","BYR","BZD","CAD","CDF","CHF","CLP","CNY","COP","CRC","CUC","CUP","CVE","CZK","DJF","DKK","DOP","DZD","EGP","ERN","ETB","EUR","FJD","FKP","GBP","GEL","GGP","GHS","GIP","GMD","GNF","GTQ","GYD","HKD","HNL","HRK","HTG","HUF","IDR","ILS","IMP","INR","IQD","IRR","ISK","JEP","JMD","JOD","JPY","KES","KGS","KHR","KMF","KPW","KRW","KWD","KYD","KZT","LAK","LBP","LKR","LRD","LSL","LTL","LVL","LYD","MAD","MDL","MGA","MKD","MMK","MNT","MOP","MRO","MUR","MVR","MWK","MXN","MYR","MZN","NAD","NGN","NIO","NOK","NPR","NZD","OMR","PAB","PEN","PGK","PHP","PKR","PLN","PYG","QAR","RON","RSD","RUB","RWF","SAR","SBD","SCR","SDG","SEK","SGD","SHP","SLL","SOS","SPL","SRD","STD","SVC","SYP","SZL","THB","TJS","TMT","TND","TOP","TRY","TTD","TVD","TWD","TZS","UAH","UGX","USD","UYU","UZS","VEF","VND","VUV","WST","XAF","XCD","XDR","XOF","XPF","YER","ZAR","ZMW","ZWD","BTC"]},"status":{"type":"string","description":"Transaction status, required for state change notifications","enum":["SUCCESS","UNKNOWN","PENDING","PAYMENT_FAILURE","PLUGIN_FAILURE","PAYMENT_SYSTEM_OFF"]},"gatewayErrorCode":{"type":"string"},"gatewayErrorMsg":{"type":"string"},"firstPaymentReferenceId":{"type":"string"},"secondPaymentReferenceId":{"type":"string"},"properties":{"type":"array","items":{"$ref":"#/definitions/PluginProperty"}},"isAdjusted":{"type":"boolean"},"adjustments":{"type":"array","items":{"$ref":"#/definitions/InvoiceItem"}},"auditLogs":{"type":"array","items":{"$ref":"#/definitions/AuditLog"}}}},"InvoiceDryRun":{"type":"object","properties":{"dryRunType":{"type":"string","enum":["TARGET_DATE","UPCOMING_INVOICE","SUBSCRIPTION_ACTION"]},"dryRunAction":{"type":"string","enum":["START_ENTITLEMENT","START_BILLING","PAUSE_ENTITLEMENT","PAUSE_BILLING","RESUME_ENTITLEMENT","RESUME_BILLING","PHASE","CHANGE","STOP_ENTITLEMENT","STOP_BILLING","SERVICE_STATE_CHANGE"]},"phaseType":{"type":"string","enum":["TRIAL","DISCOUNT","FIXEDTERM","EVERGREEN"]},"productName":{"type":"string"},"productCategory":{"type":"string","enum":["BASE","ADD_ON","STANDALONE"]},"billingPeriod":{"type":"string","enum":["DAILY","WEEKLY","BIWEEKLY","THIRTY_DAYS","SIXTY_DAYS","NINETY_DAYS","MONTHLY","BIMESTRIAL","QUARTERLY","TRIANNUAL","BIANNUAL","ANNUAL","BIENNIAL","NO_BILLING_PERIOD"]},"priceListName":{"type":"string"},"subscriptionId":{"type":"string","format":"uuid"},"bundleId":{"type":"string","format":"uuid"},"effectiveDate":{"type":"string","format":"date"},"billingPolicy":{"type":"string","enum":["START_OF_TERM","END_OF_TERM","IMMEDIATE","ILLEGAL"]},"priceOverrides":{"type":"array","items":{"$ref":"#/definitions/PhasePrice"}}}
```

Comment:
New file?

Reply:
Yes, sorry, I think it got generated while testing. I've deleted this now.